man/tpm2_createek.1.md

```diff
@@ -58,12 +58,13 @@ Refer to:
     https://trustedcomputinggroup.org/wp-content/uploads/
     TCG_IWG_Credential_Profile_EK_V2.1_R13.pdf
 
-  * **-f**, **\--format**=_FORMAT_:
 
-    Public key format.
+[pubkey options](common/pubkey.md)
+
 
 ## References
 
+
```

Comment:
spurious whitspace changes here and through out, remove them.

Reply:
Done!

Reply:
How is this done? The '+ whitespace' issue is still here. Did you amend your commit with the changes and force push back to your master branch?